### DISCUSSION TITLE
Object Detection models training

### DISCUSSION THREAD

**Discussion by koldogut** [robmarkcole/HASS-Deepstack#183] [Q&A]
I'm using the default object models in deepstack and sometimes I get false positives detecting persons (latests with a cute snowman in my garden at 04:55am, hehehe); as my home alarm is in part based on this detection i'd like to tell the model when the coincidence is true or false. Any suggestion ?

**Comment by robmarkcole** [comment]
I dont understand the question, you want a model that rejects snowmen?

**Reply by koldogut** [reply]
Hehe, no, I want the model to not identify the snowman as a person. I understand it should be done by re-training the model to get more accuracy but I don't know how.

**Comment by robmarkcole** [comment]
Please raise this for discussion on the deepstack forum, see you there
